tests/test_Galactic.py

```diff
@@ -160,19 +169,23 @@ def test_ecliptic_to_galactic(self):
             "Ecliptic to Galactic conversion for now failed with separation %.1e arcsec"
             % sep.arcsec
         )
-        assert sep < 1e-4 * u.arcsec, msg
+        assert sep < 1e-9 * u.arcsec, msg
 
         B1855_ECL = self.modelB1855.coords_as_ECL(epoch=newepoch)
         # what I get converting within astropy
         B1855_galactic_comparison = B1855_ECL.transform_to(astropy.coordinates.Galactic)
         B1855_galactic_then = utils.remove_dummy_distance(
             B1855_galactic_now.apply_space_motion(
-                new_obstime=astropy.time.Time(newepoch, format="mjd")
+                new_obstime=astropy.time.Time(newepoch, scale="tdb", format="mjd")
             )
         )
         sep = B1855_galactic_then.separation(B1855_galactic_comparison)
         msg = (
             "Ecliptic to Galactic conversion for +100d failed with separation %.1e arcsec"
             % sep.arcsec
         )
-        assert sep < 1e-6 * u.arcsec, msg
+        assert sep < 1e-9 * u.arcsec, msg
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
This isn't needed with `pytest`.

Reply:
No, but it's helpful when iterating the test in a console.

Reply:
I just use `pytest tests/test_Galactic.py`.

Reply:
(`pytest` generally provides nicer output and better control of what happens in tests. It also supports parametrized tests and various other nice things `unittest` doesn't, but even for `unittest` tests it is generally a nicer tool.)